### PR TITLE
Add a new page with information about TTT events

### PR DESCRIPTION
**Pull request by peteryates**
### Trello card

https://trello.com/c/7LARh17h/2575-events-build-about-ttt-events-page

### Context and changes

- Add the 'about TTT events' box to event listing
- Add 'about TTT events' info page

https://user-images.githubusercontent.com/128088/142186702-85d4bae4-9df4-4abb-8a76-95ec19e2cb6f.mp4

Update from the above clip, now the link on the index page is styled like a button:

Screenshot from 2021-11-17 10-57-51

### CODE REVIEW

**Review by ethax-ross — APPROVED** [review]
Looks good to me. Could probably add it to the static page actions as well?

File: app/views/teaching_events/about_ttt_events.html.erb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      to take your next step.
+    </p>
+
+    <p>
+      To attend, you need to register. Spaces are limited, so the sooner you sign
+      up, the better.
+    </p>
+
+    <p>
+      <a href="/teaching-events" class="button">Register for a Train to Teach event</a>
```
Comment (gemmadallmandfe) [inline app/views/teaching_events/about_ttt_events.html.erb:35]: Can this button go to a filtered list of events ie with TTT events only selected (this is what the user would expect?)

File: app/views/teaching_events/index.html.erb [hunk 2]
```diff
@@ -23,6 +23,19 @@
             <h2>Upcoming Train to Teach events</h2>
 
             <ol><%= render TeachingEvents::EventComponent.with_collection(@featured_events) %></ol>
+
+            <div class="teaching-events__about-ttt-banner">
+              <div class="teaching-events__about-ttt-banner--message">
+                <p>
+                  The main event. Designed by the Department for Education to
```
Comment (gemmadallmandfe) [inline app/views/teaching_events/index.html.erb:30]: Text is slightly too close to edge of pink diagonal on desktop?

File: app/views/teaching_events/about_ttt_events.html.erb [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      <li>put your questions to expert advisers, teachers and training providers</li>
+      <li>chat with current teachers and find out what a career in teaching is really like</li>
+      <li>listen to a step-by-step guide on how to get into teaching, the application process and funding your training</li>
+    </ul>
+  </article>
+
+  <%= render(
+      partial: "teaching_events/show/what-theyre-saying",
+      locals: {
+        quote: "It was exactly what I needed to give me guidance on the next steps to be taken"
```
Comment (gemmadallmandfe) [inline app/views/teaching_events/about_ttt_events.html.erb:52]:
```suggestion
        quote: "It was exactly what I needed to give me guidance on the next steps to be taken."
```

**Review by ethax-ross — APPROVED** [review]

**Pull request merged**